test/suite/test_layered23.py

```diff
@@ -353,7 +347,6 @@ def test_leader_follower(self):
         # Then advance the checkpoint and make sure everything is still good
         self.pr('advance checkpoint')
         self.disagg_advance_checkpoint(conn_follow)
-        self.check_checkpoint_follower(conn_follow, checkpoint_count)
```

Comment:
I think it would still be good to have some way to verify that we actually picked up a checkpoint.  If `disagg_advance_checkpoint` doesn't guarantee that, maybe we can look at statistics?  I'm happy if you want to push this into another ticket, as you probably want to get this change in quick.